# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Feedback - Current arm position
float64[] arm_position          # [Right_tilt, Right_lift, Right_rotate, Right_elbow]

